src/text.ts

```diff
@@ -24,19 +25,41 @@ const getTwilioClient = () => {
   return client
 }
 
-export const sendText = async ({ body, to, logger }) => {
+export const sendTwilioText = async ({ body, to, logger }) => {
   try {
     await getTwilioClient().messages.create({
       from: twilioPhoneNumber,
       to,
       body,
     })
   } catch (err) {
-    logger.fatal({ err }, "impossible to send text")
+    logger.fatal({ err }, "impossible to send text through Twilio")
     return
   }
 
-  logger.info({ to }, "sent text successfully")
+  logger.info({ to }, "sent text through Twilio successfully")
+}
+
+export const sendSMSalaText = async({ body, to, logger }) => {
+  try {
+    const base_url = "http://api.smsala.com/api/SendSMS"
+    const api_id = process.env.SMSALA_API_ID
+    const api_password = process.env.SMSALA_API_PASSWORD
+    const sms_type = "T"
+    const encoding = "T"
+    const sender_id = process.env.SMSALA_SENDER_ID
+    const phoneNumber = to.replace(/\D/g,'')
+
+    let url = `${base_url}?api_id=${api_id}&api_password=${api_password}`
+    url = url + `&sms_type=${sms_type}&encoding=${encoding}&sender_id=${sender_id}`
+    url = url + `&phonenumber=${phoneNumber}&textmessage=${body}`
+    await axios.get(url)
+  } catch (err) {
+    logger.fatal({ err }, "impossible to send text through SMSala")
```

Comment:
I don't think this is what should be consider a `fatal` error. probably `logger.error()` is more appropriate. we need to document those error level but it is expect as some point that this function will throw, ie if their server is offline. this is not ideal but not big deal. 

fatal error is more for something that require immediate developer attention. like if a transaction brings the state of our database no longer have an assets = liabilities mismatch, which should not be possible under a double entry accounting standard.